test/suites/basic.sh

```diff
@@ -126,7 +126,11 @@ test_basic_usage() {
 
   # Test container copy
   lxc copy bar foo
+  lxc config device set foo eth0 ipv4.address=192.0.2.2
+  # Test container with static IP copy
+  lxc copy foo baz
```

Comment:
Can you try starting it here and check it fails too.

Reply:
I check this in [clustering.sh](https://github.com/lxc/lxd/pull/10363/commits/7b696ff1dee5e5e409b4cf0a1845c7727ecbf6cb#diff-7cd3b6dcbb2b188abfdc6a6141d12b1f779cd4ce27356d86f1e855dd1414923a)